docs/users-guide/13-sql-parameters.md

```diff
@@ -155,6 +155,31 @@ WHERE True
   [[AND {% raw %}{{category}}{% endraw %}]]
 ```
 
+### Using an existing question as a sub-query
+
+You can use an existing question in a new query with the following variable syntax:
+
+```
+SELECT count(*)
+FROM {% raw %}{{#123}}{% endraw %}
+```
+
+This will return the number of records returned by the existing question with ID 123. A question's ID is the number at the end of the URL in your browser's location bar, after `/question/`, when viewing the question.
+
+Alternatively, you can select the target question in the sidebar, in the "Question #..." box that appears after typing `{% raw %}{{#}}{% endraw %}` in the query editor.
```

Comment:
Let's add a note here:

"The saved question you select has to be one that's based on the same database as the one you've currently selected in the native query editor."